modules/statics/src/base.ts

```diff
@@ -14,7 +14,7 @@ export const enum CoinKind {
  *
  * For example, the coins `btc` and `tbtc` both belong to the same family, `btc`.
  */
-export const enum CoinFamily {
+export enum CoinFamily {
```

Comment:
Why remove `const`?

Reply:
so this is basically so we can enumerate in our tests - const enum are not actually iterable, they're not even a thing in JS, they're like preprocessor directives in C I believe

https://stackoverflow.com/questions/28818849/how-do-the-different-enum-variants-work-in-typescript